modules/log/src/EventSubscriber/OrderEventSubscriber.php

```diff
@@ -34,6 +36,7 @@ public static function getSubscribedEvents() {
       'commerce_order.validate.pre_transition' => ['onValidateTransition', -100],
       'commerce_order.fulfill.pre_transition' => ['onFulfillTransition', -100],
       'commerce_order.cancel.pre_transition' => ['onCancelTransition', -100],
+      OrderEvents::ORDER_ASSIGN => ['onOrderAssign', -100],
```

Comment:
We don't above because they don't exist. And that seems silly if core recommends real IDs yet constants provided. I'll change to keep it consistent here.